openhands/core/config/agent_config.py

```diff
@@ -30,6 +33,7 @@ class AgentConfig(BaseModel):
     disabled_microagents: list[str] = Field(default_factory=list)
     enable_history_truncation: bool = Field(default=True)
     enable_som_visual_browsing: bool = Field(default=False)
+    codeact_enable_search_engine: bool = Field(default=False)
```

Comment:
```suggestion
    enable_search_engine: bool = Field(default=False)
```
While we use 'codeact' for some, maybe we don't have to. We don't have many agents, and tools will differ per agent anyway.

Maybe in the future we'll do this differently (configure each agent separately), for now I don't think it hurts to simplify.

Reply:
Removed the codeact_ prefix from the config name since we dont have many agents and tools will differ per agent anyway.